### DISCUSSION TITLE
Current GPU mode while using optimized mode

### DISCUSSION THREAD

**Discussion by Diego-JRH** [seerge/g-helper#1626] [Ideas]
I would be a nice feature (I think) if when you have optimized mode in the GPU, you could visually see what mode is currently using, like if you have the laptop plugged in and the optimized mode is using standard, the standard icon appears in another color or something, the same would be applied when Eco mode is active. 

I know the text says if the iGPU or both GPUs are being used, but having it also visually displayed would be nice I think. It can be also a configurable option for users to decide if they want this feature or not.

**Comment by seerge** [comment]
@Diego-JRH you already can see that

Color of optimized button / icon in tray and icon in app is green for eco and blue for standard
Also it explicitly states dGPU state in a title of GPU section

**Reply by Diego-JRH** [reply]
<img width="274" alt="image" src="https://github.com/seerge/g-helper/assets/18351723/b1dd12b0-3213-46dc-b645-7b3b5a59d519">

Yeah I know, I mean these icons. Like the optimized mode is highlighted in blue, the mode which optimized mode is using could be also highlighted in another color.

**Reply by seerge** [reply]
It will only confuse people, only 1 button in a group can be selected at a time, and it has a border :)

**Reply by Diego-JRH** [reply]
Yeah you may be right, thats why I said it could be in a different colour and a configurable option which comes turned off by default so you know when you turn it on why 2 modes are highlighted, but it's just personal preference. That's only an idea I had that I thought it would look nice.

**Comment by Diego-JRH** [comment]
<img width="302" alt="Captura de pantalla 2023-11-23 003217" src="https://github.com/seerge/g-helper/assets/18351723/65c3cd26-0f54-447a-8c29-dd0d68216a11">
<img width="302" alt="Captura de pantalla 2023-11-23 003312" src="https://github.com/seerge/g-helper/assets/18351723/ef786db0-e783-4382-896a-d38b4662f8cb">

So I ended doing this implementation by myself but for some reason, now the app has a blue dot on the left upper corner and the build produces an additional fille called GHelper.dll.config. I would be grateful to know why this happens. Thanks.

**Reply by seerge** [reply]
@Diego-JRH blue / green / red dot is just an icon added recently :) it has same color as GPU mode (so one more indication)

As for your addition, it looks interesting, and you obviously can use that for yourself if you want :)

**Reply by Diego-JRH** [reply]
Ah cool, I see it's easily customizable.
I changed it to display the ROG logo and looks good.
![image](https://github.com/seerge/g-helper/assets/18351723/b885b65b-590c-44b4-9af3-2d27741ddee7)

**Reply by Diego-JRH** [reply]
A quick question, would be possible to add a indicator like the fn lock one for num lock?

**Reply by seerge** [reply]
I'm not a fan of that, as you already have a light on keyboard for it :)

**Reply by Diego-JRH** [reply]
In my case (I have a Strix Scar 18), it doesn't have any light or indicator that shows that num block is activated, only mayus block has a light.
